third-party/cmake/BuildLuarocks.cmake

```diff
@@ -61,6 +61,15 @@ if(UNIX OR (MINGW AND CMAKE_CROSSCOMPILING))
   elseif(USE_BUNDLED_LUA)
     list(APPEND LUAROCKS_OPTS
       --with-lua=${HOSTDEPS_INSTALL_DIR})
+  else()
+    list(APPEND CMAKE_MODULE_PATH "../cmake")
```

Comment:
This is a hack.. if this makes sense in general we should move "FindLuaJit" to `shared/` there maybe?

Reply:
@justinmk 
What do you think about this?

Should we have `cmake/shared` and add it to the module path for third-party then always?

Reply:
If it saves code and makes things more understandable then yes. If it makes things more complicated then no :)

Reply:
Ok, keeping it like this then for now.